invertedai/api/drive.py

```diff
@@ -102,14 +102,16 @@ def drive(
 
     traffic_lights_states:
        If the location contains traffic lights within the supported area,
-       their current state should be provided here. Any traffic light for which no
-       state is provided will have a state generated by iai.
+       their current state can be provided here. It is legal to not provide this field, and use
+       'light_recurrent_states' to step the traffic lights. If provided, light states from 'traffic_light_states' will override
+       the original light states given by 'light_recurrent_states'.
 
     light_recurrent_states:
        Light recurrent states for all agents, obtained from the previous call to
         :func:`drive` or :func:`initialize`.
        Specifies the state and time remaining for each light group in the map.
-       If manual control of individual traffic lights is desired, modify the relevant state(s) 
+       To let iai manage all light states in the scene, simply pass 'light_recurrent_states' from the previous response of :func:`drive` here.
+       If customized control of individual traffic lights is desired, modify the relevant state(s) 
        in traffic_lights_states, then pass in light_recurrent_states as usual.
```

Comment:
Does this actually work? I thought that once you pass the light states, it's game over for the server controller.